tdrs-frontend/src/actions/requestAccess.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+import { SET_AUTH } from './auth'
+
+export const PATCH_REQUEST_ACCESS = 'PATCH_REQUEST_ACCESS'
+export const SET_REQUEST_ACCESS = 'SET_REQUEST_ACCESS'
+export const SET_REQUEST_ACCESS_ERROR = 'SET_REQUEST_ACCESS_ERROR'
+export const CLEAR_REQUEST_ACCESS = 'CLEAR_REQUEST_ACCESS'
+
+export const requestAccess = ({ firstName, lastName, stt: { id } }) => async (
+  dispatch
+) => {
+  dispatch({ type: PATCH_REQUEST_ACCESS })
+  try {
+    const URL = `${process.env.REACT_APP_BACKEND_URL}/users/set_profile/`
+    const user = { first_name: firstName, last_name: lastName, stt: { id } }
+    const { data } = await axios.patch(URL, user, {
+      withCredentials: true,
+    })
+
+    if (data) {
+      dispatch({ type: SET_REQUEST_ACCESS })
```

Comment:
This sets `requestAccess` to `true` and `loading` to `false` in the `requestAccess` redux module.